src/main/java/frc/robot/commands/TurnToHeadingWithDriving.java

```diff
@@ -56,7 +56,7 @@ public void initialize() {
   public void execute() {
     // auto aiming using PID
       double speed = m_turnHeadingPID.calculate(m_driveTrain.getHeading().getDegrees(), m_wantedDegrees);
```

Comment:
In ActiveSetShooterWithDriving, there's a clamp() around this to mmimit it to [-1.0,1.0]. Do we want that here? Or remove it from there?